packages/client-library-otel/src/instrumentation.ts

```diff
@@ -131,6 +135,10 @@ export function instrument(app: HonoLikeApp, config?: FpxConfigOptions) {
             return respondWithRoutes(webStandardFetch, endpoint, app);
           }
 
+          // NOTE - We want to report the latest routes to FPX on every request
+          //        so that we have an up-to-date list of routes in the UI
+          sendRoutes(webStandardFetch, endpoint, app);
```

Comment:
I have a couple of ideas:
* maybe we can make it configurable whether this is executed?
* maybe this function should return a promise, so we can wait until the fetch is completed? Like using `waitUntil` or if that doesn't exist we could just add it to the promsieStore

Reply:
yeah i think pushing it into the promise store would be best solution, i'll add that

Reply:
@flenter i updated to use the promise store, and added some unit tests for the exports from `routes.ts` if you wanna take a look there